std/security/ed-signer.go

```diff
@@ -0,0 +1,110 @@
+package security
+
+import (
+	"bytes"
+	"crypto/ed25519"
+	"errors"
+	"time"
+
+	enc "github.com/named-data/ndnd/std/encoding"
+	basic_engine "github.com/named-data/ndnd/std/engine/basic"
+	"github.com/named-data/ndnd/std/ndn"
+	"github.com/named-data/ndnd/std/utils"
+)
+
+const ASN1Ed25519PrivKeyPrefix = "\x30\x2e\x02\x01\x00\x30\x05\x06\x03\x2b\x65\x70\x04\x22\x04\x20"
```

Comment:
How are these constructed? 
The procedure needs to be documented in comments. 

Why are these exported?